common/Resources/ti.internal/extensions/ti/ui/activityindicator.js

```diff
@@ -0,0 +1,9 @@
+/* globals OS_ANDROID */
+
+if (OS_ANDROID) {
+	// Define constants for ActivityIndicator here for now.
+	Titanium.UI.ActivityIndicator.STATUS_BAR = 0;
```

Comment:
This doesn't look to be documented. It appears to move `Ti.UI.Android.PROGRESS_INDICATOR_*` constants to `Ti.UI.ActivityIndicator`. Should this be `Ti.UI.ProgressIndicator` instead?

Reply:
This was copied over (and split out) from he old ui.js in Android's Kroll.js code: https://github.com/appcelerator/titanium_mobile/blob/master/android/modules/ui/src/js/ui.js#L109-L113

I don't know why it was there int he first place, so maybe we can drop it?

Reply:
@sgtcoolguy, I think you're right. The constants match 1-to-1. We should probably delete these constants.